docs/topics/install/help/dns-name-body.md

```diff
@@ -4,12 +4,14 @@ The installer could not access the EdgeStack.me service to acquire a DNS name fo
 
 ## What's next?
 
-This is likely an intermittent failure of the service. Please try again later. Simply run the installer again:
+This is likely an intermittent failure of the EdgeStack.me service. Sorry about that!
+
+Please try again later. Simply run the installer again:
```

Comment:
I can suggest trying again (_without_ the "later" qualifier), which means "try again right now," but it is likely that won't help, i.e. it'll just fail again the same way. 

1. "try again later, then get in touch" -- lousy answer, sends people away
2. "try again, then get in touch" -- almost certainly won't succeed
3. "get in touch" -- big ask (high friction)

None of these three is good.

Right now we have option 1, and you're asking for option 2. Will do.

What do you think of option 3? Are there other, better options?

Reply:
I think option 2 is best. If our edgestack.me service is reliable (which it should be), then any outage is either going to be really temporary (so trying again will succeed) or really long (because it's some major failure). In the later case, no amount of retrying will work, so no need to say "later".

Reply:
Sounds fair. We'll stick to option 2, which is what's there now.